template/template_example.Rmd

```diff
@@ -69,11 +69,11 @@ Go to this [dataset's page on refine.bio](<<REFINEBIO DATASET PAGE LINK>>).
 
 Click the "Download Now" button on the right side of this screen.  
 
-<img src="screenshots/download-now.png" width=200>  
+<img src="https://raw.githubusercontent.com/AlexsLemonade/refinebio-examples/master/template/screenshots/download-now.png" width=200>  
```

Comment:
It's a bit ugly, but I think this is a good place to use permalinks to protect against broken images. The same hash should work for all links here (replacing `master`).
```suggestion
<img src="https://raw.githubusercontent.com/AlexsLemonade/refinebio-examples/fe91830c24b5d3c289efd9fc2ff3a59611e9b001/template/screenshots/download-now.png" width=200>  
```